torch/csrc/autograd/engine.h

```diff
@@ -37,7 +37,7 @@ static constexpr int CPU_DEVICE = -1;
 // As we hold mutex in every of our custom C++ autograd Node, we would
 // like to avoid TSAN complains on this when doing reentrant backwards
 // For reference, see https://github.com/google/sanitizers/issues/950
-static constexpr int MAX_DEPTH = 60;
+static constexpr int MAX_DEPTH = 40;
```

Comment:
@swolchok why was this changed?

Reply:
@albanD this is my change, see https://github.com/pytorch/pytorch/issues/49558#issuecomment-747840440
For some reason after those inlining changes we started to hit  this one on Windows (which have default stack size of 1Mb) but never on Linux/Mac (with has a larger default stack size, at least 2Mb)

Reply:
please revert this, and pull the CoinflipTLS change since that's likely the one causing the problems